test/apivalidations/flavor_test.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright 2024 The ORC Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apivalidations
+
+import (
+	"context"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	orcv1alpha1 "github.com/k-orc/openstack-resource-controller/api/v1alpha1"
+	applyconfigv1alpha1 "github.com/k-orc/openstack-resource-controller/pkg/clients/applyconfiguration/api/v1alpha1"
+)
+
+const flavorPrefix = "flavor"
+
+func flavorStub(namespace *corev1.Namespace) *orcv1alpha1.Flavor {
+	generateResouceName(flavorPrefix)
+	obj := &orcv1alpha1.Flavor{}
+	obj.Name = generateResouceName(flavorPrefix)
+	obj.Namespace = namespace.Name
+	return obj
+}
```

Comment:
nit: missing blank line below. Surprised `go fmt` didn't complain about that 🤔

Reply:
Done